### PR TITLE
Fix UserProfile serializing.

### PR DESCRIPTION
**Pull request by serhiibuniak-okta**
Related to #207

### CODE REVIEW

**Comment by csanders-git** [comment]
component[0] exists when the input is __getstate__ because the breakdown is ['','','getstate','',''] due to the split by '_'. The problem is actually component[0][0] ... Really both should be checked but you should also be doing a check like the following
```
if len(component[0]) ==0: 
```

**Comment by serhiibuniak-okta** [comment]
@csanders-git Is there any usecase to check length of `component[0]` also? This shorter variant does the trick with the length of zero too:
```py
if components[0]:
```

**Comment by csanders-git** [comment]
Given that you're doing a '.split()' before hand you'll be guaranteed that the resultant list will be of length 1 or greater, shouldn't be a need to check.

File: okta/helpers.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def to_lower_camel_case(string):
     """
     components = string.split('_')
     # lower first letter in the first component
-    components[0] = components[0][0].lower() + components[0][1:]
+    if components[0]:
+        components[0] = components[0][0].lower() + components[0][1:]
```
Comment (laura-rodriguez) [inline okta/helpers.py:43]: Do we need to check components' length before accessing?

File: tests/unit/test_helpers.py [hunk 2]
```diff
@@ -13,3 +13,5 @@ def test_to_lower_camel_case():
     assert helpers.to_lower_camel_case('CamelCaseStr') == 'camelCaseStr'
     assert helpers.to_lower_camel_case('lowerCamelCaseStr') == 'lowerCamelCaseStr'
     assert helpers.to_lower_camel_case('snake_case_str') == 'snakeCaseStr'
+    assert helpers.to_lower_camel_case('_start_with_underscore') == 'StartWithUnderscore'
+    assert helpers.to_lower_camel_case('__start_with_double_underscore') == 'StartWithDoubleUnderscore'
```
Comment (laura-rodriguez) [inline tests/unit/test_helpers.py:17]: Can we add more tests for edge cases? Are we failing gracefully?

File: okta/helpers.py (hunk 1 above)
Comment (serhiibuniak-okta) [reply okta/helpers.py:43]: No: https://docs.python.org/3.8/library/stdtypes.html#truth-value-testing

File: tests/unit/test_helpers.py (hunk 2 above)
Comment (serhiibuniak-okta) [reply tests/unit/test_helpers.py:18]: Added. The only possible fail if an argument of given method is not the string, in this case we shouldn't fail gracefully, imho

**Review by laura-rodriguez — APPROVED** [review]
